README.md

```diff
@@ -72,7 +72,7 @@ puts user.username
 ```
 
 Apart from this, API documentation is also available, at
-https://meew0.github.io/discordcr/doc/v0.1.0/.
+https://meew0.github.io/discordcr/doc/v0.4.0/.
```

Comment:
```suggestion
https://meew0.github.io/discordcr/doc/master/.
```
This should be just master so it doesn't have to be updated every version. Also, pretty much everyone uses master.

Reply:
No, we can provide links to both if we really want to. There should always be a link to the latest tag.

Reply:
How many breaking changes can we really expect to happen in the near future.
IMO linking to master is totally fine in this case.

Reply:
You can expect everything to break because we're `v0.x`.

Linking to master is fine, but it should not replace the existing latest tag link.

Reply:
Since the latest version is broken and no one is using it I don't see the point in linking to its docs.

Reply:
No, *the point* is to fix our release cycle so that the latest version *can* be used.

Our weak release cycle isn't an excuse to change versioning practices / semantics - that tags should be relied on, and `master` or otherwise for bleeding edge changes / fixes.

(Even still, v0.4 is fine with a single monkey patch you can apply in about 5 lines.)